pkg/networkmanager/ip-settings.jsx

```diff
@@ -252,7 +252,7 @@ export const IpSettingsDialog = ({ topic, connection, dev, settings }) => {
                             </FormGroup>
                             <FormGroup className="pf-m-1-col-on-sm remove-button-group">
                                 <Button variant='secondary'
-                                        isSmall
+                                        size="sm"
```

Comment:
Same question -- if it's specific to PF5, it needs to go after some "move to PF5" commit. Otherwise, if it's valid for PF4, let's split out?

Reply:
not valid in PF5.